syntrax/build.gradle

```diff
@@ -0,0 +1,12 @@
+repositories {
+    mavenLocal()
+    mavenCentral()
+    jcenter()
+}
+
+dependencies {
```

Comment:
nit: usually dependencies are sorted, implementation dependencies come fist, then come test dependencies. So please move the new dependency to line 9

Reply:
done